src/app/replayer/replayer.ml

```diff
@@ -1212,6 +1212,20 @@ let main ~input_file ~output_file_opt ~archive_uri ~set_nonces ~repair_nonces
             ~next_epoch_ledger
         in
         let log_ledger_hash_after_last_slot () =
+          let global_slots_to_avoid =
```

Comment:
Are you sure this is the right way to do this? It seems like this should be flexible (e.g. for devnet), otherwise we'll always skip these slots regardless of the network.

Reply:
Yeah. Maybe I should just use the state_hashes for comparison.